.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Operating system**
+e.g. Windows/macOS/Linux
```

Comment:
Are you sure about this wording? It might sound a bit harsh with so many question marks.

Reply:
> Are you sure about this wording? It might sound a bit harsh with so many question marks.

I certainly did not mean to sound harsh and personally do not consider the question marks to be harsh. I fully agree with your point though as everyone will read things differently. Therefore, here comes a suggestion with less question marks and more focus on examples. :)

```suggestion
<!-- Add the operating system you are running, e.g. Windows 10 Version 20H2, macOS 10.15.5, Ubuntu Linux 20.10. Also add where you got Jamulus from, e.g. from jamulus.io, from your Linux distribution or if you built from source. -->
```

Reply:
Yes. This sounds better ;-). I think we don't need to say from jamulus.io maybe we could say "downloaded it from the internet". In fact, at the moment, there's no Jamulus binary on jamulus.io

Reply:
```suggestion
<!-- Add the operating system you are running, e.g. Windows 10 Version 20H2, macOS 10.15.5, Ubuntu Linux 20.10. Also add where you got Jamulus from, e.g. if you downloaded it from the internet, if you got it from your Linux distribution or if you built from source. -->
```